src/components/ChartContainer.js

```diff
@@ -295,44 +275,73 @@ const ChartContainer = forwardRef(
     }));
 
     return (
-      <div
-        ref={container}
-        className={"orgchart-container " + containerClass}
-        onWheel={zoom ? zoomHandler : undefined}
-        onMouseUp={pan && panning ? panEndHandler : undefined}
+      <TransformWrapper
+        initialScale={1}
+        ref={transformComponentRef}
+        disabled={!(!!zoom)}
+        minScale={zoom && zoom.minZoom}
+        maxScale={zoom && zoom.maxZoom}
+        doubleClick={{
```

Comment:
Curious why we are disabling the next 3 options?

--

I think I know the answer and that is that those gestures are also programmed in the library. Curious if we should use this dependencies handling of it though and strip out the rest.

Reply:
> Curious why we are disabling the next 3 options?

I mainly disabled these options because zooming on wheel or doubleclick aren't currently registered with the percentage from the `ZoomControl` so I created this https://trainual.atlassian.net/browse/GS1-2674 subtask because getting this far has taken a bit of time and wanted to at least submit some working zoom functionality

> Curious if we should use this dependencies handling of it though and strip out the rest.

Yeah I'm not totally sure as I think this dependency has some conflicting functionality (like `panning`) that was already built from this org chart dependency

Reply:
Yeah.. I'm just curious if your new dependency handles all that better. We can come back to this later on.